packages/nimble-components/src/button/tests/button.foundation.spec.ts

```diff
@@ -0,0 +1,669 @@
+/**
+ * Based on tests in FAST repo: https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/button/button.spec.ts
```

Comment:
Can you update this to be a permalink? (same for the checkbox one)

Reply:
I intentionally went with a non-permalink because it shows the branch name and can be used to diff with the current contents. But I guess that could be misleading if we're implying that's the version we forked. Will change.